### ISSUE TITLE
SQLAlchemy в примере бота

### ISSUE THREAD

**Issue by decorator-factory** [Rhinik/vkquick#11]
1. Бот делает всего пару запросов к SQLite, и прикручивание ORM только добавляет сложности, как мне кажется.

2. SQLAlchemy -- синхронная библиотека. Для работы с SQLite есть `aiosqlite`.

**Comment by Rhinik** [comment]
Согласен, над этим стоит подумать. В качестве орм можно взять `tortoise`. Однако, пример с орм для бота, мне кажется, полезен и вполне нужен

**Comment by decorator-factory** [comment]
Пример нужен для демонстрации работы бота, а не для демонстрации работы сторонней ORM. Если пользователь захочет использовать свою любимую ORM -- пусть использует. Не хочет -- не использует. Если ему не нужно постоянное хранилище -- пусть вообще не трогает базы данных.
